services/auth/driving/web-mvc/src/main/java/nettee/auth/web/AuthCommandApi.java

```diff
@@ -122,28 +125,29 @@ public ResponseEntity<String> sendEmailVerification(@RequestBody EmailVerifySend
             description = "사용자가 이메일 인증코드를 확인합니다."
     )
     public ResponseEntity<String> verifyEmail(@RequestBody EmailVerifyRequest request) {
-        // 이메일 인증 코드 확인 로직 구현
         String token = authSignUsecase.verifyOtp(request.email(), request.otp(), request.nonce());
         return ResponseEntity.ok(token);
     }
 
-    // TODO: 비밀번호 변경 전 재인증 & 비밀번호 변경을 함께 진행할 수도 있음
-    @PostMapping("/password/verification")
+    @PostMapping("email/password/reset/send")
     @Operation(
-            summary = "비밀번호 변경 전 재인증",
-            description = "사용자가 비밀번호를 변경하기 전에 재인증을 수행합니다."
+            summary = "이메일 비밀번호 재설정 링크 전송",
+            description = "사용자의 이메일로 비밀번호를 변경할 수 있는 링크를 발송합니다."
     )
-    public void changePassword(@RequestBody PasswordVerifyRequest passwordVerifyRequest) {
-        // 비밀번호 변경 로직 구현
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public ResponseEntity<String> sendPasswordResetEmail(@RequestBody PasswordForgotRequest request) {
+        authSignUsecase.sendPasswordResetEmail(request.email());
```

Comment:
`NO_CONTENT` 응답이므로 `void` 또는 `ResponseEntity<Void>`를 사용할 수 있어 보입니다! 👍 

누락된 작업 같아서 제안드립니다.

```suggestion
    @ResponseStatus(HttpStatus.NO_CONTENT)
    public ResponseEntity<Void> sendPasswordResetEmail(@RequestBody PasswordForgotRequest request) {
        authSignUsecase.sendPasswordResetEmail(request.email());
        return ResponseEntity.noContent().build();
```